cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -1718,6 +1718,20 @@ public Map<ProgramRunId, RunRecordDetail> getActiveRuns(ApplicationReference app
     return getRuns(Range.singleton(prefix), ProgramRunStatus.ALL, Integer.MAX_VALUE, null, null);
   }
 
+  /**
+   * Get number of active runs of the given program irrespective of the versions of the programs.
+   * Active runs means program run with status PENDING, STARTING, RUNNING, SUSPENDED or STOPPING.
+   *
+   * @param programRef {@link ProgramReference} of the given program
+   * @return int number of active runs
+   */
+  public int getNumberOfActiveRunsAcrossVersions(ProgramReference programRef)
```

Comment:
Please follow naming similar to io.cdap.cdap.internal.app.store.AppMetadataStore#getProgramTotalRunCount.

Reply:
Done